js-core/homeworks/phoneApp/src/main.js

```diff
@@ -0,0 +1,111 @@
+function PhoneApp() {
+    this.storage = [];
+}
+
+PhoneApp.prototype.checkNumber = function(numberToCheck) {
+    const toStringThePredictNumber = numberToCheck.toString();
```

Comment:
are you sure it work correct with numbers ?

how would you store such cases ?
```bash
099-13-15-100
```
or similar with zeros ?